docs/usage/configuration/config_documentation.md

```diff
@@ -3815,6 +3815,8 @@ Sub-options for this setting include:
 * `system_mxid_display_name`: set the display name of the "notices" user
 * `system_mxid_avatar_url`: set the avatar for the "notices" user
 * `room_name`: set the room name of the server notices room
```

Comment:
Should this section note that changing these settings won't take affect until a new notice is sent into the room?

Reply:
I added a small note.